### DISCUSSION TITLE
Polymorphic Allocator

### DISCUSSION THREAD

**Discussion by hemalbavishi** [eclipse-iceoryx/iceoryx#1399] [Q&A]
Hello,

Is there any plan to implement PMR to allocate memory during initialization based on configurable element such as number of subscribers, number of publishers etc...

Many Thanks!

Best Regards,
Hemal Bavishi

**Comment by elBoberido** [comment]
@hemalbavishi our memory management is not ideal and there are some discussions like #911 or #1169 but nothing is decided yet.
